src/CommonGLPI.php

```diff
@@ -590,6 +590,20 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
         return '';
     }
 
+    public static function getSectorizedDetails(): array
+    {
+        return [];
+    }
+
+    public static function getHeaderParameters(): array
+    {
+        return [
+            static::getTypeName(\Session::getPluralNumber()),
+            $_SERVER['PHP_SELF'],
```

Comment:
In fact, this parameter is useless. You can use a `''` value.
We cannot change the `Html::header()` signature because it is used in too many places, plugins included, and it would be a pain for everyone to handle the signature change.
```suggestion
            '',
```